commit dedacd6d3612ec1473c14eff7135de90f40c9298 (refs/changes/88/5388/2)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2019-01-18T13:51:53-08:00 (9 months ago)
    
    AGENT-997 hermes-actor should use sdcnode
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/CHANGES.md b/CHANGES.md
index 3bf6895..7411c35 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,9 @@
 # sdc (SDC ops core zone) Changelog
 
+## 1.6.0
+
+- AGENT-997 update hermes to sdcnode (v6.15.1)
+
 ## 1.5.2
 
 - MANTA-4030 Drop pinned lru-cache top-level dep. The release or lru-cache@4.1.5
diff --git a/deps/hermes b/deps/hermes
index 3702adf..6744fa4 160000
--- a/deps/hermes
+++ b/deps/hermes
@@ -1 +1 @@
-Subproject commit 3702adf367e3ea2a6b0c27889a0e9e0b4ac93ebc
+Subproject commit 6744fa48829270d50ee8331a58f372b9aae5cc12
diff --git a/package.json b/package.json
index e8e8154..84d5d12 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "sdc",
     "description": "SmartDataCenter 'sdc' zone to hold ops/admin tools",
-    "version": "1.5.2",
+    "version": "1.6.0",
     "author": "Joyent (joyent.com)",
     "private": true,
     "dependencies": {
